src/NetworkEntities.js

```diff
@@ -62,21 +62,11 @@ class NetworkEntities {
 
     if (this.hasEntity(networkId)) {
       this.entities[networkId].components.networked.networkUpdate(entityData);
-    } else if (this.isFullSync(entityData)) {
+    } else if (entityData.isFirstSync) {
```

Comment:
I think this should be `isFullSync` not `isFirstSync` I think there are other cases we may want to send full syncs, such as after ownership transfer.